docs/using-kubelinter.md

```diff
@@ -0,0 +1,72 @@
+# Using KubeLinter
+
+You can run KubeLinter both locally and on your CI systems.
+
+## Running locally
+
+After you've [installed KubeLinter](README.md#installing-kubelinter), use the
+`lint` command and provide:
+
+<!-- tabs:start -->
+
+#### ** Kubernetes **
+- The path to your Kubernetes `yaml` file:
+  ```bash
+  kube-linter lint /path/to/yaml-file.yaml
+  ```
+- The path to a directory containing your Kubernetes `yaml` files:
+  ```bash
+  kube-linter lint /path/to/directory/containing/yaml-files/
+  ```
+
+#### ** Helm **
+The path to a directory containing the `Chart.yaml` file:
+```bash
+kube-linter lint /path/to/directory/containing/Chart.yaml-file/
+```
+
+<!-- tabs:end -->
+
+## KubeLinter commands
+
+This section covers kube-linter command syntax, describes the command
+operations, and provides some common command examples.
+
+### Commands syntax
+
+Use the following syntax to run KubeLinter commands:
```

Comment:
Not for this PR, but the library we use for command-line parsing actually supports generating markdown documentation, so we can embed that in the docs site eventually:
https://github.com/spf13/cobra/blob/master/doc/md_docs.md

Reply:
That'd be great.